collection_manager/containers/docker/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+python /collection_manager/collection_manager/main.py \
+  $([[ ! -z "$COLLECTIONS_PATH" ]] && echo --collections-path=$COLLECTIONS_PATH) \
+  $([[ ! -z "$RABBITMQ_HOST" ]] && echo --rabbitmq-host=$RABBITMQ_HOST) \
```

Comment:
Do we need these lines for the kubernetes/helm integration ?

Reply:
These lines will convert environment variables into command line parameters to send to the collection manager script. While it is not strictly necessary, it is a good way of encapsulating the command line syntax of the script and isolating it from the Kubernetes template that launches the docker container.

Reply:
Ok I see, that makes sense for a service which runs alone on its docker image.